### DISCUSSION TITLE
choose a network interface to bind

### DISCUSSION THREAD

**Discussion by bart3nder** [XTLS/Xray-core#1518] [General]
Hi!
Is there an option to choose a network interface to bind in our configs (Vmess/Vless/Trojan)? Like the way we do in shadowsocks with `-i` flag e.g. `ss-server -c config.json -i eth9`, which pass the traffic through an specified interface.

**Comment by cross-hello** [comment]
See here(please translate the document to English before reading )
https://github.com/XTLS/Xray-core/pull/1494

**Comment by bart3nder** [comment]
Thanks for the answer, Everything works fine except I'm facing WebRTC IP leak when I bind to another interface (like VPN). It there any solution for that?

**Comment by cross-hello** [comment]
😅 Did you have written correct outbound routing rules?

Jan 12, 2023 00:30:02 deBartender ***@***.***>:

**Comment by bart3nder** [comment]
I'm Using the routing rule from the thread you sent:
`"settings": {"domainStrategy": "AsIs"},`
Ans as I said it didn't work. Is there something else I should use? Sorry I'm a bit newbie to this 😅
